src/adapters/polkadot.ts

```diff
@@ -174,7 +189,42 @@ class BasePolkadotAdapter extends BaseCrossChainAdapter {
 
     const accountId = this.api?.createType("AccountId32", address).toHex();
 
-    const [dst, acc, ass] = supportsV0V1Multilocation(this.api)
+    const doesSupportV0V1 = supportsV0V1Multilocation(this.api);
+
+    // to statemine
+    if (to === "statemine" || to === "statemint") {
+      const dst = {
+        interior: { X1: { ParaChain: toChain.paraChainId } },
+        parents: 0,
+      };
+      const acc = {
+        interior: {
+          X1: {
+            AccountId32: {
+              id: accountId,
+              network: doesSupportV0V1 ? "Any" : undefined,
```

Comment:
Do we still need the v0v1 support? If not, it'd be cleaner to remove it. But also happy to turn that into an issue to do later